docs/Linuxbrew.md

```diff
@@ -0,0 +1,99 @@
+![Linuxbrew logo](https://linuxbrew.sh/images/linuxbrew-256x256.png)
+
+# Linuxbrew
+
+The Homebrew package manager may be used on Linux and Windows 10+, using [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about), as well as macOS. Homebrew is referred to as Linuxbrew when running on Linux or Windows. It can be installed in your home directory and does not require root access. Recent versions of *glibc* and *gcc* may be installed in your home directory on an older distribution of Linux. A Linuxbrew installation does not use any libraries provided by your host system, except *glibc* and *gcc* if they are sufficiently current.
```

Comment:
First sentence:
> Besides macOS, Homebrew also supports Linux and Windows 10+ (_via_ its [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about)).

Reply:
I'm inclined to remove the mention of macOS, as its stating the obvious to say that Homebrew runs on macOS.

Reply:
Maybe: "The Homebrew package manager may also be used on Linux and Windows 10+, using Windows Subsystem for Linux (WSL)."

Reply:
> I'm inclined to remove the mention of macOS, as its stating the obvious to say that Homebrew runs on macOS.

Agreed.

> does not require root access

does not use the root user

> may be installed in your home directory

may be installed in your Homebrew prefix (or some similar wording that still works if Linuxbrew is installed outside the home directory)

> if they are sufficiently current.

if they are new enough

Reply:
>> does not require root access.

> does not use the root user.

went with `does not require sudo.`

Reply:
@sjackman I'm being pedantic but I think it's worth noting when things like `sudo` are mentioned that _after installation_ it doesn't _use it_ (ever) rather than doesn't _require_ it (which sounds like it will if it can)

Reply:
Went with…

It can be installed in your home directory, in which case it does not use sudo.
…
The installation script installs Linuxbrew to `/home/linuxbrew/.linuxbrew` using sudo if possible and in your home directory at `~/.linuxbrew` otherwise. Linuxbrew does not use sudo after installation.